frontend/src/Editor/Editor.jsx

```diff
@@ -112,22 +113,6 @@ class EditorComponent extends React.Component {
       currentLayout: 'desktop',
       deviceWindowWidth: 450,
       appDefinition: this.defaultDefinition,
-      currentState: {
-        queries: {},
-        components: {},
-        globals: {
-          theme: { name: props.darkMode ? 'dark' : 'light' },
-          urlparams: JSON.parse(JSON.stringify(queryString.parse(props.location.search))),
```

Comment:
Seems like we're not updating the store with these values. It should be updated during the mount

Reply:
Yes, Done